src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -35,7 +35,7 @@ CAC.Map.Templates = (function (Handlebars, Utils) {
         var source = [
             '<div class="block-detail">',
                 '<h3>{{ d.name }}</h3>',
-                '<h5>20 minutes away</h5>',
+                '<h5 class="distanceMinutes">{{ d.durationMinutes }} minutes away</h5>',
```

Comment:
May want to change this to "distance-minutes" to match the style of the other classes.